Core/CMakeLists.txt

```diff
@@ -186,6 +186,7 @@ add_executable(transformix_exe
   ${InstallFilesForExecutables}
 )
 set_target_properties(transformix_exe PROPERTIES OUTPUT_NAME transformix)
+target_compile_definitions(transformix_exe PRIVATE ELX_CMAKE_VERSION="${CMAKE_VERSION}")
```

Comment:
Another possibility might be to do `set_source_files_properties`, specifically for elxMainExeUtilities.cxx